intro.md

```diff
@@ -258,45 +290,43 @@ Note: analogy: you can bring your project to Black&Decker, and then you'll have
 <!--s-->
 # Security nightmare
 
-- Need to allow other people on your platform for proximity
-- And they can use whatever software they want on the platform
+- You share the platform with others <!-- .element: class="fragment"-->
+- You can use whatever they want <!-- .element: class="fragment"-->
+- Streams have access control lists <!-- .element: class="fragment"-->
+- Tenants are separated <!-- .element: class="fragment"-->
```

Comment:
What was the idea of your changes? Before, the bullets give two required platform features that the trainer can use to build an urgency for proper management of access control. The first two new bullets have the same 'outcome', but without any reasoning why it was necessary. The last two new bullets are mitigating the 'nightmare' which seems like they don't belong in the same list.

Reply:
Good point, removed the last two bullets.
The first two bullets attempt to make the sharing and freedom explicit. Should they change?